packages/backend-for-frontend/src/services/producerKeychainService.ts

```diff
@@ -16,8 +12,7 @@ import { decorateKey } from "./clientService.js";
 import { getSelfcareCompactUserById } from "./selfcareService.js";
 
 export function producerKeychainServiceBuilder(
-  apiClients: PagoPAInteropBeClients,
-  selfcareUsersClient: SelfcareV2UsersClient
+  apiClients: PagoPAInteropBeClients
 ) {
   const { authorizationClient } = apiClients;
```

Comment:
You can destructure `selfcareV2UserClient` here as well as done in clientService.ts